### DISCUSSION TITLE
NetBox without the IPAM

### DISCUSSION THREAD

**Discussion by ColtMercer** [netbox-community/netbox#6135] [Q&A]
I would like to use NetBox but I do not want to use the IPAM. I have always heard that to be possible however I cannot assign an IP address to a device unless I have full IPAM configured. In addition, the primary IP address of real world network devices is a Loopback interface. If I create an IP Address and try to assign it to a device as the primary IP the only option I have is to assign it to the physical interfaces, not a virtual. This cumbersome process is not ideal... If you can help can you please try to make sense out of why the system was designed this way? This makes easy steps in other DCIM extremely cumbersome in NetBox.

**Comment by devon-mar** [comment]
> I do not want to use the IPAM

You don't need to use all the IPAM features (prefixes, VLANs, etc). You can just use it to track IP addresses.

> the only option I have is to assign it to the physical interfaces, not a virtual

What do you mean by this? You can assign an IP address to an interface of type `Virtual`  and then set it as the primary IP for the device.
